src/controllers/AccommodationController.js

```diff
@@ -71,6 +71,23 @@ class AccommodationController {
       return Response.errorMessage(req, res, 'Server Error', 500);
     }
   }
+
+  /**
+  * Travel Admin can be able to create accommodation facility
+   * @static
+   * @param {object} req  request object
+   * @param {object} res response object
+   * @memberof AccommodationController
+   * @returns {object} data
+   */
+  static async getAccommodations(req, res) {
+    try {
+      const accommodations = await getAccommodations(req);
+      return Response.successMessage(req, res, 'Accommodation/s successfully fetched', accommodations, 200);
```

Comment:
I think if you want to specify accommodation/accommodations, simply write `accommodation(s)`